src/mlpack/methods/preprocess/preprocess_split_main.cpp

```diff
@@ -73,6 +80,7 @@ PARAM_DOUBLE_IN("test_ratio", "Ratio of test set; if not set,"
     "the ratio defaults to 0.2", "r", 0.2);
 
 PARAM_INT_IN("seed", "Random seed (0 for std::time(NULL)).", "s", 0);
+PARAM_FLAG("shuffle_data", "Avoid shuffling and spliting the data.", "S");
```

Comment:
```suggestion
PARAM_FLAG("shuffle_data", "Avoid shuffling and splitting the data.", "S");
```

Also, this indicates that if `shuffle_data` is true, we avoid shuffling and splitting the data, which isn't true. I think the description should be `Shuffle and split the data`

Reply:
Oops I'm sorry, I didn't read further, you've complemented it. However, the name still seems misleading to me. I would intuitively assume that `shuffle_data` means that it shuffles data if true.

Reply:
Changed it `no_shuffle` , I agree but I ain't good at naming 😉